.github/workflows/build-version-docs.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will build gradio and create the version's docs file when a new stable version has been released 
+
+name: Create Version's Docs
+on:
+  workflow_dispatch:
+  pull_request:
+    types: closed
+    branches:
+      - main
+    paths:
+      - 'gradio/version.txt'
+
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.x'
+    - name: Install requirements
+      run: python -m pip install -r website/homepage/requirements.txt 
+    - name: Check new pypi version
+      run: cd website && python check_version.py
+  build-docs:
+    needs: check-version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install requirements
+        run: python -m pip install -r website/homepage/requirements.txt 
+      - name: Install gradio
+        run: python -m pip install gradio -U
```

Comment:
We install from pip here right? And the reason is so that we do not use the local front-end code?

Reply:
Not sure what you mean by local front-end code. 

We install from pip the latest released version because that's where we want to generate these docs from. Part of the docs comes from gradio as a module and not form the `/website` dir. For example `gradio.generate_documentation()`. 

But on what will show up in `/docs/main` we install gradio from the repo and not from pip (this happens inside the Dockerfile) 

Lmk if this isn't what you mean